js/app.js

```diff
@@ -0,0 +1,163 @@
+const a = 4;
+const b = 53;
+const c = 57;
+const d = 16;
+const e = 'Kevin';
+
+console.log(a < b);
+console.log(c > d);
+console.log('Name' === 'Name');
+console.log(a < b < c);
```

Comment:
this evaluates to 'true' but not for the reason that you might think.
it's looking for `a + b === c`